app.py

```diff
@@ -9,22 +9,32 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-import json
+
+import json 
 import os
-from typing import List
+from typing import List   
```

Comment:
Duplicate import of 'PromptTemplate' from langchain, which may cause confusion and code redundancy.